src/linked_hash_map.rs

```diff
@@ -469,15 +513,23 @@ impl<A: fmt::Debug + Hash + Eq, B: fmt::Debug> fmt::Debug for LinkedHashMap<A, B
     }
 }
 
-unsafe impl<K: Send, V: Send> Send for LinkedHashMap<K, V> {}
+impl<K: Hash + Eq, V: PartialEq, S: HashState> PartialEq for LinkedHashMap<K, V, S> {
+    fn eq(&self, other: &LinkedHashMap<K, V, S>) -> bool {
+        self.len() == other.len() && self.iter().zip(other.iter()).all(|(l, r)| l == r)
```

Comment:
I'd say yes, since that's the difference between a `LinkedHashMap` and a `HashMap`.